flink-cdc-cli/src/test/resources/definitions/pipeline-definition-full.yaml

```diff
@@ -53,3 +53,7 @@ pipeline:
   name: source-database-sync-pipe
   parallelism: 4
   enable-schema-evolution: false
+
+flink-config:
```

Comment:
flink-config parallelism and cdc-confg parallelism, how is the priority divided here?

Reply:
Currently, if parallelism exists both in the Flink config and CDC config, the Flink config will override the CDC config.

Reply:
OK